### PR TITLE
Art Creator Feature 

### LINKED ISSUE #116: Create art creator feature
Whatever you type on the left hand side box shows up on the right side.
This task already has two PRs associated with it. Check [here](https://github.com/Real-Dev-Squad/website-www/pull/145) and [here](https://github.com/Real-Dev-Squad/website-www/pull/143).

[Figma Link](https://www.figma.com/file/bbT6ouwsrjYzs6G4XDSn3a/CSS-Art?node-id=0%3A1)

### PR DESCRIPTION
**Pull request by abhinav1590**
closes #116 
Example output :-
image

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -487,3 +487,67 @@ footer {
...
+  outline: none;
+}
+
+.container {
+  text-align: center;
+  margin-top: 20px;
+}
+
+.button {
+  background: linear-gradient(90deg, #e30162 25%, #ed579c 90%);
```
Comment (Vividh25) [inline css/main.css:543]: Could we get rid of the linear gradient maybe? It's looking out of place.

**Review by Vividh25 — CHANGES_REQUESTED** [review]
Great work ! Just one minor suggestion.

File: css/main.css (hunk 1 above)
Comment (abhinav1590) [reply css/main.css:543]: Fixing it right away

**Review by Vividh25 — APPROVED** [review]
Great work! Approving ✔

File: art-feature.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <head>
+    <title>Art Feature</title>
+    <link rel="icon" href="/img/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+  <body>
+    <h1 class="welcome-title">Your Art</h1>
+    <div class="editor">
+      <div class="html-container">
+        <input type="text" placeholder="Thinking is the enemy of creativity." />
```
Comment (harshith-venkatesh) [inline art-feature.html:12]: Nice placeholder :)

File: art-feature.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <textarea id="html-code"></textarea>
```
Comment (harshith-venkatesh) [inline art-feature.html:13]: Textarea expands now both horizontally and vertically right... Should we maximize the width horizontally

File: css/main.css [hunk 4]
```diff
@@ -487,3 +487,66 @@ footer {
...
+}
+
+#output {
+  width: 300px;
+  height: 328px;
+}
+input {
+  text-align: center;
+  border: none;
+  outline: none;
```
Comment (harshith-venkatesh) [inline css/main.css:534]: Outline: none is not advisable considering accessibility.
I will share you a link in discord channel, you can refer that and could try making relevant changes

File: css/main.css [hunk 5]
```diff
@@ -487,3 +487,66 @@ footer {
...
+
+.button {
+  background-color: #db0979;
+  color: white;
+  cursor: pointer;
+  font-weight: bold;
+  border: none;
+  margin-top: 15px;
+  border-radius: 5px;
+  padding: 15px 15px;
```
Comment (harshith-venkatesh) [inline css/main.css:550]: We can mention 15px once here, it applies to all

File: art-feature.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+      </div>
+      <div class="output-container">
+        <iframe id="output"></iframe>
+      </div>
+    </div>
+    <div class="container">
+      <button class="button">Submit for 500 Dineros</button>
```
Comment (harshith-venkatesh) [inline art-feature.html:20]: What is the purpose of the button here... I have not observed any ID attached to it or any function being called onclick of this... Please point me out to the code if I had missed it

File: css/main.css (hunk 4 above)
Comment (harshith-venkatesh) [reply css/main.css:534]: https://betterprogramming.pub/a11y-never-remove-the-outlines-ee4efc7a9968

File: art-feature.html (hunk 6 above)
Comment (abhinav1590) [reply art-feature.html:20]: The purpose of this button is not yet defined but I am sure about it but it's something related to the backend of this feature😅

**Pull request merged**